CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Fixed number of related products in mobile view
```

Comment:
This is more of a side-effect of your change, but I wouldn't say this is the actual change. Maybe something along the lines of:

```markdown
### Fixed
- `isMobile` prop passed to `ProductList` by `RelatedProducts` component would always be `false` due to object spreading order.
```

Reply:
Done. Thanks!